src/lib.rs

```diff
@@ -640,10 +640,11 @@ impl Build {
 
     /// Set CUDA C++ support.
     ///
-    /// Enabling CUDA will pass the detected C/C++ toolchain as an argument to
-    /// the CUDA compiler, NVCC. NVCC itself accepts some limited GNU-like args;
-    /// any other arguments for the C/C++ toolchain will be redirected using
-    /// "-Xcompiler" flags.
+    /// Enabling CUDA will invoke the CUDA compiler, NVCC. While NVCC accepts
+    /// most common compiler flags, e.g. `-std=c++17`, some project-specific
+    /// flags might have to prefixed with "-Xcompiler" flag, for example as
+    /// `.flag("-Xcompiler").flag("-fpermissive")`. Consult Nvidia
+    /// documentation for further information.
```

Comment:
I think it'd be great if we could also link to it.

```suggestion
    /// `.flag("-Xcompiler").flag("-fpermissive")`. See the Nvidia
    /// documentation for more information.
```